src/@next/components/atoms/CartSummaryCosts/CartSummaryCosts.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { commonMessages } from "@temp/intl";
+
+import * as S from "./styles";
+
+interface CartSummaryCostsProps {
+  subtotalPrice: React.ReactNode;
```

Comment:
As stated below, I believe applying styles and display logic for costs should be handled here, instead of passing nodes.